### DISCUSSION TITLE
Edit width of a column in a table to view data better

### DISCUSSION THREAD

**Discussion by KernOil** [Budibase/budibase#2179] [Help] [answered]
Hi Guys, 
I am having trouble with being able to read the long text column in my table. is there a way I can resize the column to display more info?

![image](https://user-images.githubusercontent.com/80058238/127531338-3b7d1a32-7f5a-4f9c-aa12-36ff047dc1df.png)

any help would be great thank you!

**Comment by joebudi** [comment, marked as answer]
hey @AndrewJRamay 

There is no way to read it within a table - we plan to make changes to this.

An alt approach would be to create a card (with a repeater), and 'manually' add your components.

**Reply by KernOil** [reply]
ok ill look into this Thank you
